charts/osm-arc/values.yaml

```diff
@@ -0,0 +1,29 @@
+# Default values for osm-arc.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+OpenServiceMesh:
+    ignoreNamespaces: "kube-system azure-arc arc-osm-system"
+
+osm:
+  OpenServiceMesh:
+    enableDebugServer: true
+    enablePermissiveTrafficPolicy: true
+    enableFluentbit: true
+    fluentBitImage:
+      name: fluentbit-logger
+      registry: "mcr.microsoft.com/oss/fluent"
```

Comment:
You also need to update the [envoy image](https://github.com/openservicemesh/osm/blob/main/charts/osm/values.yaml#L11) and the [osm-controller image](https://github.com/openservicemesh/osm/blob/main/charts/osm/values.yaml#L6-L7).

Reply:
Still missing the envoy image